lib/report/index.html.ejs

```diff
@@ -1,742 +1,723 @@
+            <path
+              d="M177.235 169.156L138.632 132.074L107.952 144.704L93.2929 130.623L228.213 75.9894L180.953 214.829L166.06 200.523L177.235 169.156ZM183.738 150.815L199.254 106.778L156.348 124.505L183.738 150.815Z"
+              fill="white" />
+            <path
+              d="M136.138 135.974L175.036 173.339L208.275 137.452L221.883 150.524L136.914 242.26L123.306 229.189L162.984 186.35L124.087 148.986L84.4087 191.824L70.8005 178.752L155.769 87.0157L169.378 100.088L136.138 135.974Z"
+              fill="white" />
+          </svg>
+        </div>
+        <!-- title and timestamp -->
+        <div>
+          <h1 style="margin-bottom: 4px" id="report-name">
+            Artillery Report
+          </h1>
+          <div id="timestamp"></div>
+        </div>
+
+      </div>
+    </header>
+    <main>
+      <!-- tabs -->
+      <div class="app-heading bg-black-default relative mb-9 pt-4 border-b border-gray-700 px-4">
+        <div class=" container tabs">
+          <div id="charts-tab" class="tab active">Charts</div>
+          <div id="json-tab" class="tab">JSON Output</div>
+        </div>
+      </div>
+
+      <div class="container">
+        <!-- summary blocks -->
+        <div class="app-test-stats w-full">
+          <ul class="flex flex-wrap space-x-3.5" id="summary-blocks">
+          </ul>
+        </div>
+        <div class="row" style="margin-top: 20px; align-items: flex-start;">
+          <div id="charts-col" class="col-lg-10 space-y-4">
+            <div class="rounded border-gray">
+              <!-- counters summary -->
+              <div id="counters-title" class="chevron-down p-3">
+                <span>
+                  <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor"
+                    class="bi bi-chevron-right" viewBox="0 0 16 16">
+                    <path fill-rule="evenodd"
+                      d="M4.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L10.293 8 4.646 2.354a.5.5 0 0 1 0-.708z" />
+                  </svg>
+                </span>
+                <div style="margin-left: 1rem">Counters</div>
+              </div>
+              <table id="customCountersTable" class="w-full"></table>
+            </div>
+            <div id="errorsChartsRow" class="row">
+              <div class="col-lg-12">
+                <h4><i class="fas fa-chart-area"></i> Errors at intervals</h4>
+                <canvas id="errorsAtIntervalsChart"></canvas>
+              </div>
+            </div>
+            <div class="row">
+              <div class="col-lg-12">
+                <!-- charts -->
+                <div id="customCountersAtIntervalsChart" class="space-y-4"></div>
+              </div>
+            </div>
+          </div>
+          <div class="col-lg-2 overflow-y-scroll" style="max-height: 100vh;" id="index-tree"></div>
```

Comment:
Also here there's a double scroll when everything is expanded...

https://user-images.githubusercontent.com/606977/173047907-987399aa-6dc2-4226-ab41-5f76e03fd8fa.mov

Reply:
This is intentional as some links flow past the edge when everything is expanded. I can remove overflow-auto for index or add a border to show the double scroll boundary